WebRoot/css/zm.css

```diff
@@ -5033,6 +5033,20 @@ If you feel compelled to have a background color, please do so on a per skin lev
 	margin:26px 24px;
 }
 
+.ZmSharedFileRemoved {
+	text-decoration: line-through;
+}
+
+.ZmSharedFilePreview {
+	color: blue;
+	background: none;
+	border: none;
+	margin: 0;
+	padding: 0;
+	cursor: pointer;
+	font-size: 14px;
```

Comment:
we have variable UserFontSize-large defined, any reason to not use it?

Reply:
used the said variable.